target/linux/ar71xx/base-files/lib/upgrade/platform.sh

```diff
@@ -647,6 +647,7 @@ platform_check_image() {
 		return 0;
 		;;
 	# these boards use metadata images
+	dap-1330-a1|\
```

Comment:
I moved the board identifier down so I don't have to add another return type.
If needed I can move it.